qcodes/tests/drivers/test_lakeshore_325.py

```diff
@@ -0,0 +1,29 @@
+from hypothesis import given
+import hypothesis.strategies as st
+from qcodes.instrument_drivers.Lakeshore import Model_325
+
+STATUS = Model_325.Model_325_Sensor.Status
+
+
+@given(
+    st.lists(st.sampled_from(list(STATUS)), 1, 5, unique=True).map(sorted)
+)
+def test_decode_sensor_status(list_of_codes):
+    """
+    The sensor status is one of the status codes, or a sum thereof. Multiple
+    status are possible as they are not necessarily mutually exclusive.
+    The static method 'decode_sensor_status' in the Model_325_Sensor class can
+    decode the status into status code(s).
+    """
+    codes = [code.name.replace('_', ' ') for code in list_of_codes[::-1]]
+    codes_message = ', '.join(codes)
+    sum_of_codes = STATUS(sum(list_of_codes))
```

Comment:
this is incorrect. `decode_sensor_status` takes an `int` as its `sum_of_codes` argument, not `Status` type, hence in the test `decode_sensor_status` should be called with an `int`, not a `Status` instance.
```suggestion
    sum_of_codes = int(sum(list_of_codes))
```

Reply:
Good catch!